### PR TITLE
release: sourcegraph@5.2.1

### PR DESCRIPTION
**Pull request by keegancsmith**
This pull request is part of the Sourcegraph 5.2.1 release.

* [Release batch change](https://sourcegraph.sourcegraph.com/organizations/sourcegraph/batch-changes/release-sourcegraph-5.2.1)
* [Tracking issue](https://github.com/sourcegraph/sourcegraph/issues/57319)

### Test plan

CI checks in this repository should pass, and a manual review should confirm if the generated changes are correct.

### CODE REVIEW

File: docker-compose/embeddings/embeddings.docker-compose.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ services:
   #
   embeddings:
     container_name: embeddings
-    image: 'index.docker.io/sourcegraph/embeddings:insiders@sha256:0e4f28307fb942d57a7ef0963e53e9fcf99c1d34dfaf2ba5345d9faecb5954f1'
+    image: 'index.docker.io/sourcegraph/embeddings:insiders@sha256:6bfffb3f0d15a4f71d70989656a35d617021574b9081304602e310436ce9f3f9'
```
Comment (keegancsmith) [inline docker-compose/embeddings/embeddings.docker-compose.yaml:10]: cc @stefanhengl also using insiders tag here, but atleast in this repo executors uses a tagged version.

**Review by camdencheek — APPROVED** [review]

**Review by keegancsmith — COMMENTED** [review]
LGTM

File: docker-compose/embeddings/embeddings.docker-compose.yaml (hunk 1 above)
Comment (keegancsmith) [reply docker-compose/embeddings/embeddings.docker-compose.yaml:10]: hah, the CI over here fails due to the insiders tag. I wonder why previous releases worked. the CI must of become more strict. I am gonna manually add a tag and test that it exists and has the same hash.

Comment (keegancsmith) [reply docker-compose/embeddings/embeddings.docker-compose.yaml:10]: Well this is a good catch, I'll open a PR for the othe rrepo which is also using insiders. But this sha256 is not for the released tag.

**Pull request merged**